package org.jetbrains.kotlinworkshop.advanced._2Interop;

public class UsingThrow {


    public void usingThrow() {
        _4ThrowsKt.throwsException("Something");
    }
}
